### PR TITLE
[feature] cwd option

### PR DESCRIPTION
**Pull request by KostiantynPopovych**
This PR is adding an ability to specify the path where the check should be performed.

### CODE REVIEW

File: tests/engine/cwd-option.spec.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { depngn } from '../../src';
+
+describe('cwd option', () => {
+	it('possible to perform the check in the existing directory passing a relative path', async () => {
+		const checkResult = await depngn({ version: '18.0.0', cwd: '.' });
+		expect(checkResult).not.toBeNull();
```
Comment (etagwerker) [inline tests/engine/cwd-option.spec.ts]: @KostiantynPopovych Can we actually expect something a bit more specific? This is the same expectation as specified in the next test case.

**Review by etagwerker — COMMENTED** [review]
@KostiantynPopovych This looks very interesting. Just had one question in the tests you added.

File: tests/engine/cwd-option.spec.ts (hunk 1 above)
Comment (KostiantynPopovych) [reply tests/engine/cwd-option.spec.ts]: @etagwerker, thank you for reviewing! And it is a great point. I'll take a look at what we can expect here)

Comment (etagwerker) [reply tests/engine/cwd-option.spec.ts]: @KostiantynPopovych sounds good. Did you get a chance to improve this part?

Comment (KostiantynPopovych) [reply tests/engine/cwd-option.spec.ts]: @etagwerker just pushed my update, thank you.

**Review by etagwerker — APPROVED** [review]
@KostiantynPopovych looks good, thanks!

**Pull request merged**